classes/class-pmpro-subscription.php

```diff
@@ -148,6 +148,38 @@ public function __construct( $subscription = null ) {
 		}
 	}
 
+	/**
+	 * Get a property of this object.
+	 *
+	 * @since TBD
+	 *
+	 * @param  string $property The property to get.
+	 * @return mixed  The property value.
+	 */
+	public function __get( $property ) {
```

Comment:
Should we be doing get_*() methods to match how get_next_payment_date() etc work?